### ISSUE TITLE
GRPC loading proto error

### ISSUE THREAD

**Issue by jamiemac87** [nestjs/nest#712]
## I'm submitting a...

<pre><code>
[ ] Regression 
[x] Bug report
[x] Feature request
[ ] Documentation issue or request
[ ] Support request => Please do not submit support request here, instead post your question on Stack Overflow.
</code></pre>

## Current behavior
I have a proto package in `node_modules`  that I want to load using the following client options:
```
export const grpcClientOptions: ClientOptions = {
  transport: Transport.GRPC,
  options: {
    url: '0.0.0.0:50051',
    package: 'ordering',
    protoPath: './node_modules/@flytio/protos/ordering/ordering.proto'
  },
}
```

my proto imports another proto like so:

```
syntax = "proto3";

package ordering;

import "google/protobuf/timestamp.proto";

...

```

The google protobufs are also within the npm package and reside in a sibling folder to my ordering proto:
```
/ordering
../ordering.proto
/google
../protobuf
../../timestamp.proto
```

when running `yarn start` i get the following error:
```
Invalid .proto definition (file not found?)
Error: Invalid .proto definition (file not found?)
```

If i remove the import everything works as expected.

## Expected behavior
Is there a way to load a proto specifying a root as you would do if loading via the gprc library

```
this.proto = grpc.load({ root: "./node_modules/@flytio/protos", file: "ordering/ordering.proto" }).ordering;
```

I believe this would solve the issue.

## Environment

<pre><code>
Nest version: 5.0.0

 
For Tooling issues:
- Node version: XX  
- Platform:  

Others:

</code></pre>

**Comment by kamilmysliwiec** [comment]
Fixed in `v5.0.1` (`root` property is now available). Leaving opened to remember about updating the docs

**Comment by kamilmysliwiec** [comment]
Let's track this issue here https://github.com/nestjs/docs.nestjs.com/issues/102
